buildkite/clone_repositories.sh

```diff
@@ -19,7 +19,10 @@ for dir in "${repositories[@]}" ; do
         mkdir -p "$dir"
         for f in ./* ; do
             case "$f" in
-                ./.git) ;;
+                ./.git)
+                    # for some commands a real "git" repository is required
+                    cp -r "$f" "$dir"
+                    ;;
```

Comment:
How does this work, isn't the clone performed further down the script?

Reply:
No this is for the initial repository from which the PR was created and buildkite currently automatically checks out for us.

Reply:
I see.